common/src/main/java/com/wynntils/core/features/FeatureManager.java

```diff
@@ -273,6 +273,7 @@ private void registerFeature(Feature feature) {
                 System.exit(1);
             }
         } catch (Throwable exception) {
+            // FIXME!!!
```

Comment:
I assume a crashing init did fail to init all other features. I saw this behavior too.

Reply:
Oops, I forgot this! That should not go in this PR; it was supposed to be a reminder to me to report this to the crash dump telemetry in a separate PR.